apps/dash-mapd-demo/app.py

```diff
@@ -72,11 +109,16 @@ def generate_dest_choro(dd_select, start, end):
 
     choro_query = f"SELECT AVG(depdelay) AS avg_delay, {state_col} AS state FROM {table} WHERE dep_timestamp BETWEEN '{start_f}' AND '{end_f}' GROUP BY {state_col}"
 
+    new_con = None
```

Comment:
what's the purpose of `new_con` in comparison to `init_con` 
it might make the code a little more readable to give new_con a more descriptive name or add a comment there explaining the need for multiple connections vars

Reply:
Fixed